arm9/source/game/cert.c

```diff
@@ -1,21 +1,319 @@
 #include "cert.h"
-#include "ff.h"
+#include "disadiff.h"
 
-u32 LoadCertFromCertDb(u64 offset, Certificate* cert, u32* mod, u32* exp) {
-    Certificate cert_local;
-    FIL db;
-    UINT bytes_read;
+typedef struct {
+    char magic[4]; // "CERT"
+    u8 unk[4]; // afaik, always 0
+    u8 used_size[4]; // size used after this header
+    u8 garbage[4]; // literally garbage values
+} PACKED_STRUCT Certsdb_partition_header;
```

Comment:
Better (and matching the current style): CertsDbPartitionHeader

Reply:
Fixed in commit 809d031